data_check_tool_python/src/coseal_reader.py

```diff
@@ -783,3 +948,115 @@ def remove_features(self):
                     
         return True
         
+    def remove_algorithm_features(self):
+        '''
```

Comment:
please add a docstring explain what this function is for

Reply:
I updated the docstring